Amazon.IonHashDotnet/Serializer.cs

```diff
@@ -277,7 +285,7 @@ private int GetLengthLength(byte[] bytes)
             return 0;
         }
 
-        private (byte, byte[]) ScalarOrNullSplitParts(IonType type, bool isNull, byte[] bytes)
+        private (byte, byte[]) ScalarOrNullSplitParts(IonType type, dynamic ionValue, byte[] bytes)
```

Comment:
based on usage within this method, it appears `ionValue` is expected to be a `SymbolToken`.  can we declare it as such?

Reply:
Updated parameter  name to symbolToken and type: SymbolToken?